ahachul_backend/src/main/kotlin/backend/team/ahachul_backend/api/member/domain/entity/MemberEntity.kt

```diff
@@ -46,4 +46,16 @@ class MemberEntity(
                         )
                 }
         }
+
+        fun changeNickname(nickname: String) {
+                this.nickname = nickname
+        }
+
+        fun changeGender(gender: GenderType) {
+                this.gender = gender
+        }
+
+        fun changeAgeRange(ageRange: String) {
+                this.ageRange = ageRange
+        }
```

Comment:
```suggestion
fun memberUpdate(nickname: String, gender: GenderType, ageRange: String) {
         this.nickname = nickname
         ...
  }
```
이런식으로 비즈니스 메서드 하나만 만들어서 한번에 받는게 좋을 것 같아용